.github/workflows/release.yaml

```diff
@@ -21,6 +21,10 @@ jobs:
     if: startsWith(github.ref, 'refs/tags/v') && github.repository == 'open-policy-agent/gatekeeper'
     timeout-minutes: 45
     steps:
+      - name: Cleanup disk
```

Comment:
Will it be better to run it at the end?

Reply:
No, since agents are not persistent and they start with almost full disk.